rest-service/manager_rest/utils.py

```diff
@@ -228,3 +230,30 @@ def current_tenant():
 
 def set_current_tenant(tenant):
     g.current_tenant = tenant
+
+
+def unzip(archive, destination=None, logger=None):
+    if not destination:
+        destination = tempfile.mkdtemp()
+    if logger:
+        logger.debug('Extracting zip {0} to {1}...'.
+                     format(archive, destination))
+    with zipfile.ZipFile(archive, 'r') as zip_file:
+        zip_file.extractall(destination)
+    return destination
+
+
+def files_in_folder(folder, name_pattern='*'):
+    files = []
+    for item in os.listdir(folder):
```

Comment:
note that listdir returns both files and dirs inside the given dir.
so this func may return dirs that match the pattern as well.
consider add is_file() check.